Gemfile

```diff
@@ -48,13 +50,13 @@ group :development, :test do
   gem 'factory_girl_rails'
   gem 'pry-rails'
   gem 'rspec-rails', '~> 3.3.0'
+  gem 'apivore', git: 'https://github.com/westfieldlabs/apivore.git'
```

Comment:
wondering if we should tag a certain commit when referencing these libs via git? (helpful to know at what point in git history we were working as a way of versioning)